tern/formats/default/generator.py

```diff
@@ -113,8 +114,14 @@ def get_layer_info_list(layer):
         file_level_licenses = ", ".join(layer_file_licenses_list)
 
     for package in layer.packages:
+        pkg_licenses = package.pkg_license
+        if package.pkg_format == 'deb':
+            pkg_licenses = wrap(", ".join(package.pkg_licenses)) or ['']
```

Comment:
Let's add a `width` value to this wrap as well as a comment. I think 45 should be sufficient so the table line is not too ridiculously long:

```
            # If many pkg_licenses, wrap the line for the table at 45 chars
            pkg_licenses = wrap(", ".join(package.pkg_licenses),
                                                width=45) or ['']

```